### ISSUE TITLE
[BUG] My t2+ shinies dont appear shiny

### ISSUE THREAD

**Issue by Sundayy8** [pagefaultgames/pokerogue#2700]
**Screenshots / Videos**

**Device**

**Additional context**

**Comment by flx-sta** [comment]
@Sundayy8 please fill out the template/form before we can proceed. Alternative you can head to our discord channel for bug reports:
https://discord.com/channels/1125469663833370665/1126338176659763210

**Comment by flx-sta** [comment]
Closing this in favor of #2701
